Tools/DeployApps/BaasClient.cs

```diff
@@ -699,6 +717,17 @@ private static class Schemas
                 roles = new[] { _defaultRoles }
             };
 
+            private static object GenericFlxBaasRule(string differentiator, string collectionName) => new
+            {
+                collection = collectionName,
+                database = $"FLX_{differentiator}",
+                roles = new[] { _flxDefaultRoles }
+            };
+
+            public static object AsymmetricObjectWithAllTypesBaasRule(string differentiator) => GenericFlxBaasRule(differentiator, "AsymmetricObjectWithAllTypes");
+
+            public static object HugeSyncAsymmetricObjectBaasRule(string differentiator) => GenericFlxBaasRule(differentiator, "HugeSyncAsymmetricObject");
```

Comment:
We probably don't need these explicit functions, we can use the common one directly. Also, we should use `nameof` here instead of hardcoded strings.